presto-parser/src/main/java/com/facebook/presto/sql/tree/ExpressionTreeRewriter.java

```diff
@@ -555,6 +549,32 @@ public Expression visitFunctionCall(FunctionCall node, Context<C> context)
             return node;
         }
 
+        // Since OrderBy contains list of SortItems, we want to process each SortItem's key, which is an expression
+        private OrderBy rewriteOrderBy(OrderBy orderBy, Context<C> context)
```

Comment:
Side note: I was looking at this class and saw that normal OrderBy isn't rewritten here -- shouldn't it be? (cc @sopel39)